### ISSUE TITLE
Classify the generated image

### ISSUE THREAD

**Issue by JikaiHou** [eriklindernoren/PyTorch-GAN#41]
I run your code about cgan and the generated image looks quite perfect. However, when I tried to classified the generated image by another network (Resnet18), the predicted label is always 'eight'. Is this a common feature of cgan?

**Comment by eriklindernoren** [comment]
That's interesting. :) Haven't read anything similar to your experience regarding cGAN. I've seen similar biases before when the values of the test image are not scaled similarly to the training data. But that's not the case here?
